### PR TITLE
Add createdViaDiscovery

### PR DESCRIPTION
**Pull request by itdove**
https://github.com/open-cluster-management/backlog/issues/13246
Signed-off-by: Dominique Vernier <dvernier@redhat.com>

### CODE REVIEW

File: pkg/collectors/managedclusterinfo.go [hunk 1]
```diff
@@ -260,3 +257,14 @@ func createManagedClusterListWatchWithClient(client dynamic.Interface) cache.Lis
 		},
 	}
 }
+
+func getCreateVia(client dynamic.Interface, mci *mciv1beta1.ManagedClusterInfo, mc *mcv1.ManagedCluster) string {
```
Comment (hanqiuzh) [inline pkg/collectors/managedclusterinfo.go:261]: createdvia or createvie 😂

Comment (itdove) [reply pkg/collectors/managedclusterinfo.go:261]: ;-)

**Pull request merged**